ui/shared/components/panel/variants/Pathogenicity.jsx

```diff
@@ -70,6 +70,9 @@ const Pathogenicity = ({ variant }) => {
               formatName={snakecaseToTitlecase}
             />,
           )}
+          {variant.clinvar.gold_stars != null &&
+            <Rating icon="star" defaultRating={variant.clinvar.gold_stars} maxRating={4} />
```

Comment:
on the server side you use `goldStars` and here you use `gold_stars` so this won't work. Can you also post a quick screenshot of what this looks like?